### PR TITLE
some fixes, see commit messages

### PR DESCRIPTION
**Pull request by xHeaven**

### CODE REVIEW

File: src/Routing/ResolvedRoute.php [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
 use stdClass;
 
-class ResolvedRoute
+readonly class ResolvedRoute
 {
     public function __construct(
-        public readonly Route $route,
-        public readonly stdClass $args
+        public Route    $route,
+        public stdClass $args
```
Comment (CommandString) [inline src/Routing/ResolvedRoute.php:11]: Hmm why did you drop the read-only?

File: tools/HttpExceptions/GenerateClasses.php [hunk 2]
```diff
@@ -19,11 +19,11 @@ abstract class Code {
...
 
     $namespace = "Tnapf\Router\Exceptions";
-    
+
     $parsedPhrase = preg_replace("/[^a-zA-Z]/", "", $code->phrase);
     $className = "Http".$parsedPhrase;
 
-    ob_start(); 
+    ob_start();
     echo "<?php\n"; ?>
```
Comment (CommandString) [inline tools/HttpExceptions/GenerateClasses.php:27]: But I need this so the code is PHP

File: src/Routing/ResolvedRoute.php (hunk 1 above)
Comment (xHeaven) [reply src/Routing/ResolvedRoute.php:11]: I've made the class itself to be `readonly`, class properties will inherit this initializer.

Comment (CommandString) [reply src/Routing/ResolvedRoute.php:11]: Ohh, I didn't even see that. My fault

File: tools/HttpExceptions/GenerateClasses.php (hunk 2 above)
Comment (xHeaven) [reply tools/HttpExceptions/GenerateClasses.php:27]: Can you explain? I've just removed excess spaces here.
The only notable change I've made is the `define()` to `const` conversion, which does the exact same thing actually, but it's the new standard.

Comment (CommandString) [reply tools/HttpExceptions/GenerateClasses.php:27]: It looks like you removed echo <?php

Comment (CommandString) [reply tools/HttpExceptions/GenerateClasses.php:27]: Oh wait, nevermind. Seems github mobile is just being stupid

**Review by CommandString — APPROVED** [review]

**Pull request merged**